components/Layout/Layout.js

```diff
@@ -1,27 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { Box, Cell, utils } from 'ui-kit';
+import { Box } from 'ui-kit';
 import { Footer, Header, SEO } from 'components';
 
-const DEFAULT_CONTENT_WIDTH = utils.rem('1100px');
-
 function Layout(props = {}) {
   return (
     <>
       <SEO title={props.title} />
       <Box display="flex" flexDirection="column" height="100vh">
         <Header />
-        <Box flexGrow="1">
-          <Cell
```

Comment:
Are these `Layout`-level changes to support full-bleed hero images?
I took a 10min stab at what would be involved for it, and gave up after realizing I'd have to decouple the `<Cell>` from `<Layout>` and have to touch all the page containers.

Actually, on second thought — looks like this was to properly theme the background color at the page level?
Either way, I like these changes!

We could almost collapse the common props used for `<Cell>` down as defaults, to reduce the cluttery boilerplate...

Reply:
Cell/layout changes are for full bleed background colors. I think I already decoupled all the Cell/Layout stuff per page containers and contentLayout. When this is really ready to review that would be a good things to look for, it will be pretty obvious if something is broken.

I think that is a good idea to push those defaults into Cell 👍